sections/fido.tex

```diff
@@ -1,9 +1,31 @@
+This heterogeneity
+%even in the relatively small field of solar physics
+increases the effort required by scientists to get the data they need. 
+In order to address this issue \sunpypkg provides a single and powerful data search and retrieval interface which can interface with all data providers. 
+Referred to as \Fido, it provides a unified data search and download interface for user that simplifies and homogenizes search and retrieval to multiple data providers. 
+\Fido currently supports the Virtual Solar Observatory (VSO), the Joint Science Operations Center (JSOC) (see Section \ref{sec:drms}) and a number of individual data providers that make their data available via web-accessible resources such as HTTP websites (RHESSI, SDO-EVE, NOAA GOES soft X-ray flux, PROBA2-LYRA and NOAA sunspot number prediction) and FTP sites (NOAA sunspot number, Nobeyama Radioheliograph).
 
-The principle interface between \sunpy and observational solar physics data is the \Fido\ data search and retrieval module. The purpose of \Fido is to present a unified data search and download interface to the user that simplifies and homogenizes the use of multiple data providers.  At present, \Fido interfaces with the Virtual Solar Observatory (VSO), the Joint Science Operations Center (JSOC) (see Section \ref{sec:drms}) and a small number of data providers that make their data available via web-accessible resources such as HTTP websites (RHESSI, SDO-EVE, NOAA GOES soft X-ray flux, PROBA2-LYRA and NOAA sunspot number prediction) and FTP sites (NOAA sunspot number, Nobeyama Radioheliograph).
+A \Fido search can include multiple instruments, and can query all available data providers with a single query.  
+Search queries make use of search tokens (e.g. instrument, time range, wavelength) which can be joined using Boolean operators enabling complex search queries to be constructed easily. 
+The result of a query can be inspected and edited before retrieval. 
+\Fido makes use of asynchronous download streams significantly improving download speeds. 
 
-A \Fido search can include multiple instruments, and can query all available data providers.  For example, a single search request can be issued that will query both the LYRA and RHESSI providers individually.  Data downloads via the \code{Fido.fetch} method can be split up into multiple parallel streams, improving download speeds.  \Fido also handles failed data downloads: the output from \code{Fido.fetch} flags failed downloads, and passing that output back in to \code{Fido.fetch}\ allows \Fido to attempt to download those files again.
+%Data downloads via the \code{Fido.fetch} method can be split up into multiple parallel streams, improving download speeds.  \Fido also handles failed data downloads: the output from \code{Fido.fetch} flags failed downloads, and passing that output back in to \code{Fido.fetch}\ allows \Fido to attempt to download those files again.
```

Comment:
This meant to be commented out?

Reply:
I'm not sure that it makes sense to compare the diffs. You should look at the latest overleaf to see if I did anything wrong.

Reply:
Yes, it is meant to be commented out.